orcid-model/src/main/resources/client_3.0_dev1/samples/client-3.0_dev1.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    =============================================================================
+
+    ORCID (R) Open Source
+    http://orcid.org
+
+    Copyright (c) 2012-2014 ORCID, Inc.
+    Licensed under an MIT-Style License (MIT)
+    http://orcid.org/open-source-license
+
+    This copyright and license information (including a link to the full license)
+    shall be included in its entirety in all copies or substantial portion of
+    the software.
+
+    =============================================================================
+
+-->
+<client:client xmlns:client="http://www.orcid.org/ns/client"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.orcid.org/ns/client ../client-2.1.xsd ">    
```

Comment:
Hey @bobcaprice, please check all xml sample files, in many of them there are references to the 2.1 XSD, please change them to 3.0_dev1 XSD.

Thanks